mongodbatlas/resource_backup_compliance_policy.go

```diff
@@ -534,3 +571,7 @@ func expandDemandBackupPolicyItem(d *schema.ResourceData) *matlas.PolicyItem {
 
 	return &onDemand
 }
+
+func printWarningMessageForFirstLastNameFields() {
+	log.Printf("[WARN] Note: Starting from v1.14.0 authorized_user_first_name and authorized_user_last_name will become mandatory. Please update your script accordingly.")
```

Comment:
to make this more visible for the end user, have you considered adding a diagnostic with a warning severity?

Reply:
Yes that is the option I had in mind for adding warnings. It does have the benefit that users see the warning each time they run a terraform command similar to deprecation messages. If we go for defining attributes as required this will no longer be necessary.

Reply:
Correct, it will no longer be necessary as per the conversation I had with Zuhair.